### PR TITLE
Billing/pricing doc simplification

### PR DESCRIPTION
**Pull request by zuluecho9**
Some prep work for QBP. Work done: 

Delete 'billing UI' doc and move that info to main 'NR pricing' doc
Put the 'data ingest' and 'billable users' docs under a 'Billing factors' subcategory
Some assorted language improvements

### CODE REVIEW

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx [hunk 1]
```diff
@@ -14,37 +14,55 @@ redirects:
...
 ---
 
-New Relic has two primary billing factors: ingested data and billable users. In this doc, we'll explain how pricing works, what you get for free, and other details. 
+import accountsUsageUi from 'images/accounts_screenshot-full_usage-ui.webp'
+
+The New Relic pricing model is simple and relies on two primary billing factors: how much data you ingest and how many billable users you have. We'll explain where to find your billing UI, how our pricing plan works, what you get for free, and other details. 
 
 <Callout variant="tip">
-This doc is about our newer usage-based pricing model, and not our original pricing model. You're probably on our newer pricing model, but if you're not sure, see [Pricing models](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-pricing-models).
+A small number of older New Relic customers are on an older pricing model. For docs on that, see [Pricing models](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-pricing-models).
```
Comment (jeff-colucci) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx:26]: "Docs on that" seems a little informal. Maybe just "for information" or some variant thereof?

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx [hunk 2]
```diff
@@ -14,37 +14,55 @@ redirects:
...
 </Callout>
 
-## Where to start [#start]
+## Not yet a customer? [#start]
+
+If you're not yet a New Relic customer and want an overview of how our pricing works, we recommend checking out our main [pricing page](https://newrelic.com/pricing). If you want more in-depth details about how our pricing works, keep reading. 
+
+## The billing and usage UI [#usage-ui]
+
+If you're a New Relic customer, the best way to understand your billing is to go to the usage UI ([direct link to UI](https://one.newrelic.com/admin-portal/consumption-usage)). To find that, go to **[one.newrelic.com](https://one.newrelic.com)**, click the [user menu](/docs/accounts/accounts-billing/general-account-settings/intro-account-settings), then click **Administration**. You'll land on the **Plan & usage** UI page, which shows an overview of your usage.
```
Comment (jeff-colucci) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx:35]: Do we need the (direct link to UI) part? Can that link just be the "usage UI"?

Comment (zuluecho9) [reply src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx]: i don't really know. sometimes i think it's unclear if that's just linking to another doc or what

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx [hunk 3]
```diff
@@ -14,37 +14,55 @@ redirects:
...
 
 ### Pricing editions [#editions]
 
-Besides determining the cost of your billable users, the editions (Standard, Pro, Enterprise) gate access to different features and limits (for example, different limits for synthetic monitor checks, and access to [Data Plus](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus), which gives you HIPAA- and FedRAMP-compliant data). For more on editions, see the [pricing page](https://newrelic.com/pricing). 
+Besides determining how much your full platform users cost, your edition (Standard, Pro, or Enterprise) gates access to assorted features and limits (for example, different limits for synthetic monitor checks, and access to [Data Plus](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus), which gives you HIPAA- and FedRAMP-compliant data). For more about edition differences, see our [pricing page](https://newrelic.com/pricing).
```
Comment (jeff-colucci) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx:56]: "gates" seems a bit awkward to read to me. Maybe just "provides" or something like that?

**Pull request merged**